### PR TITLE
Pinot Data Integration doc page

### PR DESCRIPTION
**Pull request by HussienSayed14**
## Description

Adding Doc page for Pinot Data Integration. (Apache-Pinot.mdx) 

**Fixes** #(4905)

- [x] 📄 This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
Recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by HussienSayed14** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by martyna-mindsdb** [comment]
Related to issue https://github.com/mindsdb/mindsdb/issues/4905

File: docs/data-integrations/Apache-Pinot.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+---
+title: Apache Pinot
+sidebarTitle: Apache Pinot
+---
+
+Apache Pinot Handler
```
Comment (martyna-mindsdb) [inline docs/data-integrations/Apache-Pinot.mdx:7]: Add `##` at the beginning of this line to make it a header.

File: docs/data-integrations/Apache-Pinot.mdx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This is the implementation of the Apache Pinot handler for MindsDB.
+
+Apache Pinot
```
Comment (martyna-mindsdb) [inline docs/data-integrations/Apache-Pinot.mdx:10]: Add `##` at the beginning of this line to make it a header.

File: docs/data-integrations/Apache-Pinot.mdx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Apache Pinot is a real-time distributed OLAP database designed for low-latency query execution even at extremely high throughput. Apache Pinot can ingest directly from streaming sources like Apache Kafka and make events available for querying immediately.
+https://www.startree.ai/what-is-apache-pinot
+
+Implementation
```
Comment (martyna-mindsdb) [inline docs/data-integrations/Apache-Pinot.mdx:14]: Add `##` at the beginning of this line to make it a header.

File: docs/data-integrations/Apache-Pinot.mdx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+The required arguments to establish a connection are,
+
+host: the host name or IP address of the Apache Pinot cluster
+broker_port: the port that the Broker of the Apache Pinot cluster is running on
+controller_port: the port that the Controller of the Apache Pinot cluster is running on
+path: the query path
+Usage
+In order to make use of this handler and connect to a Apache Pinot cluster in MindsDB, the following syntax can be used,
+
+CREATE DATABASE pinot_datasource
```
Comment (martyna-mindsdb) [inline docs/data-integrations/Apache-Pinot.mdx:26]: Enclose it in a code block like this:

> \```sql
> CREATE DATABASE pinot_datasource
> WITH
> engine='pinot',
> parameters={
>     "host":"localhost",
>     "broker_port": 8000,
>     "controller_port": 9000,
>     "path": "/query/sql",
>     "scheme": "http"
> };
> \```

File: docs/data-integrations/Apache-Pinot.mdx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+parameters={
+    "host":"localhost",
+    "broker_port": 8000,
+    "controller_port": 9000,
+    "path": "/query/sql",
+    "scheme": "http"
+};
+Now, you can use this established connection to query your database as follows,
+
+SELECT * FROM pinot_datasource.example_tbl
```
Comment (martyna-mindsdb) [inline docs/data-integrations/Apache-Pinot.mdx:38]: Enclose it in a code block like this:

> \```sql
> SELECT * FROM pinot_datasource.example_tbl;
> \```

File: docs/data-integrations/Apache-Pinot.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+
```
Comment (martyna-mindsdb) [inline docs/data-integrations/Apache-Pinot.mdx:1]: Remove this empty line.

**Comment by martyna-mindsdb** [comment]
@HussienSayed14 
Please see the review comments above and make the changes in this PR. Thanks.

**Comment by HussienSayed14** [comment]
@martyna-mindsdb  I Have made these changes could please review if there anything esle that need to be updated or improved.
Thanks.

**Comment by martyna-mindsdb** [comment]
@HussienSayed14 
You didn't include `sql` in the code blocks. It should be as below:

> \```sql
> CREATE DATABASE pinot_datasource
> WITH
> engine='pinot',
> parameters={
>     "host":"localhost",
>     "broker_port": 8000,
>     "controller_port": 9000,
>     "path": "/query/sql",
>     "scheme": "http"
> };
> \```

> \```sql
> SELECT * FROM pinot_datasource.example_tbl;
> \```

**Comment by HussienSayed14** [comment]
@martyna-mindsdb  Done, Thanks for your effort.
If there anything else please inform me.

**Pull request merged**